saleor/product/models.py

```diff
@@ -207,6 +207,7 @@ class Product(SeoModel, ModelWithMetadata, ModelWithExternalReference):
         null=True,
         on_delete=models.SET_NULL,
     )
+    recalculate_discounted_price = models.BooleanField(default=False)
```

Comment:
Similar here, for consistency we can consider changing the name to `discounted_price_dirty`

Reply:
sure, if we go with `dirty` convention this one also needs rename